src/main/java/uk/gov/companieshouse/psc/delta/mapper/KindMapper.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.companieshouse.psc.delta.mapper;
+
+import consumer.exception.NonRetryableErrorException;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.delta.PscDeleteDelta.KindEnum;
+
+@Component
+public class KindMapper {
+
+    private KindMapper() {
```

Comment:
As a component this should not have a private constructor and its method should not be static. We should inject the KindMapper into any classes which depend on it so we can mock its behaviour in a test.

Reply:
Done, comment added to explain why it has an empty constructor but it is a component